modules/oe_theme_content_project/templates/field--node--extra-field-oe-theme-content-project-funding--oe-project.html.twig

```diff
@@ -0,0 +1,48 @@
+{#
+/**
+ * @file
+ * Theme override for Funding extra field plugin.
+ *
+ * Available variables:
+ * - attributes: HTML attributes for the containing element.
+ * - label_hidden: Whether to show the field label or not.
+ * - title_attributes: HTML attributes for the title.
+ * - label: The label for the field.
+ * - multiple: TRUE if a field can contain multiple items.
+ * - items: List of all the field items. Each item contains:
+ *   - attributes: List of HTML attributes for each item.
+ *   - content: The field item's content.
+ * - entity_type: The entity type to which the field belongs.
+ * - field_name: The name of the field.
+ * - field_type: The type of the field.
+ *
+ * @see template_preprocess_field()
+ */
+#}
+{% if items.0.content['#items'] %}
+{% set _items = items.0.content['#items'] %}
+{% else %}
+{% set _items = [] %}
+{% endif %}
+
+<div>
```

Comment:
`<div class="ecl-col-12">`

Reply:
This will change the block alignement
<img width="1232" alt="columns" src="https://user-images.githubusercontent.com/5982573/86336095-abdc4800-bc4f-11ea-9d54-6f35a59792f6.png">

Reply:
Hmmm, It isn't clear how markup should be organised. 

It is Event markup:
<img width="1009" alt="Screenshot 2020-07-02 at 11 44 38" src="https://user-images.githubusercontent.com/1183951/86337359-ad127280-bc59-11ea-824e-e92b8e5797b6.png">

For the Project top I've wrapped all blocks by div with id:
<img width="601" alt="Screenshot 2020-07-02 at 11 47 06" src="https://user-images.githubusercontent.com/1183951/86337505-da5f2080-bc59-11ea-8dd5-81efd8c8a367.png">

Not sure that there is a sense in empty div.

Reply:
For consistency I would suggest to use `<div id="project-funding" class="ecl-row">` but not sure that it is correct.

Reply:
I added the id but the class breaks the layout.